kernel/sigtools.h

```diff
@@ -302,6 +312,7 @@ struct SigMap
 
 	inline void add(Wire *wire) { return add(RTLIL::SigSpec(wire)); }
 
+	// Lazy version of add (doesn't promote if const)
```

Comment:
I think this comment is misleading, this is usually used as a representative lookup that returns the representative by updating the passed argument in-place. I'm not quite sure where the `add` overload that takes just a single SigBit is used or useful for, but the other `add` overloads don't do anything similar.

Reply:
Changed to `// Modify bit to its representative`, does that make sense?